Can't deploy package

Hi,

Following the instructions when I try to run mvnw clean package I get a maven error for a non-resolvable parent - see errors below. Any ideas?

```
 ./mvnw clean package
/jpetstore-6
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.mybatis:jpetstore:6.0.3-SNAPSHOT: Could not find artifact org.mybatis:mybatis-parent:pom:29-SNAPSHOT and 'parent.relativePath' points at no local POM @ line 22, column 11
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.mybatis:jpetstore:6.0.3-SNAPSHOT (/jpetstore-6/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.mybatis:jpetstore:6.0.3-SNAPSHOT: Could not find artifact org.mybatis:mybatis-parent:pom:29-SNAPSHOT and 'parent.relativePath' points at no local POM @ line 22, column 11 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
ubuntu@ip-172-31-30-28:/jpetstore-6$ ./mvnw cargo:run -P tomcat85
/jpetstore-6
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.mybatis:jpetstore:6.0.3-SNAPSHOT: Could not find artifact org.mybatis:mybatis-parent:pom:29-SNAPSHOT and 'parent.relativePath' points at no local POM @ line 22, column 11
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.mybatis:jpetstore:6.0.3-SNAPSHOT (/jpetstore-6/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.mybatis:jpetstore:6.0.3-SNAPSHOT: Could not find artifact org.mybatis:mybatis-parent:pom:29-SNAPSHOT and 'parent.relativePath' points at no local POM @ line 22, column 11 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```

--- a/pom.xml
+++ b/pom.xml
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.mybatis</groupId>
     <artifactId>mybatis-parent</artifactId>
-    <version>29-SNAPSHOT</version>
+    <version>29</version>
     <relativePath />
   </parent>
 

